src/javaharness/java/arcs/android/demo/service/NotificationRenderer.java

```diff
@@ -20,17 +22,22 @@
   private static final String DATA_FIELD = "data";
   private static final String TITLE_FIELD = "title";
   private static final String TEXT_FIELD = "text";
+  private static final String HANDLER_FIELD = "handler";
   private static final String OUTPUT_SLOT_ID_FIELD = "outputSlotId";
   private static final String CHANNEL_ID = "ArcsNotification";
 
   private final Context context;
+  private final PortableJsonParser jsonParser;
+
+  private static final String TAG = "Arcs";
```

Comment:
TAG is usually the class name:

private static final String TAG = NotificationRenderer.class.getSimpleName();

Reply:
done, that's what i thought, but...
we have 2 other `TAG`s that are "Arcs" - should these be changed as well? 
it is convenient to do |grep Arcs on the logcat, and how do you filter the logs?